src/internal/auth/cf_auth_middleware.go

```diff
@@ -72,14 +72,9 @@ type promqlErrorBody struct {
 }
 
 func (m CFAuthMiddlewareProvider) Middleware(h http.Handler) http.Handler {
-	router := mux.NewRouter()
-
+	router := chi.NewRouter()
 	router.HandleFunc("/api/v1/read/{sourceID:.*}", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
I don't think that including the `:.*` regex there does anything for us, should we remove it?

Reply:
feel free.

Reply:
Done